.drone.yml

```diff
@@ -18,7 +18,30 @@ steps:
   - echo "HOME FOLDER = " $HOME
 
 ---
+kind: pipeline
+type: exec
+name: integration-tests
 
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: integration-tests
+  commands:
+  - mkdir bin
+  - export PATH="$PATH:$PWD/bin"
+  - curl -sSL "https://github.com/docker/compose/releases/download/1.24.1/docker-compose-$(uname -s)-$(uname -m)" -o bin/docker-compose
```

Comment:
there's no docker compose installed?

Reply:
No, it doesn’t. At least not available for drone executing user. We could install it and save time downloading it every time tests run.